driver-async/src/test/functional/com/mongodb/async/client/JsonPoweredCrudTestHelper.java

```diff
@@ -108,6 +111,19 @@ BsonDocument getOperationResults(final BsonDocument operation, @Nullable final C
         }
     }
 
+    private String createMethodName(final String name, final String object) {
+        StringBuilder builder = new StringBuilder();
+        builder.append("get");
+        if (!object.isEmpty() && !object.equals("collection")) {
```

Comment:
We only inject the object name if its set in the json file and not the default - so we now have `getAggregateResult` and `getDatabaseAggregateResult`